cpp/src/arrow/compute/kernels/aggregate_basic.cc

```diff
@@ -378,26 +378,25 @@ namespace internal {
 namespace {
 
 const FunctionDoc count_doc{"Count the number of null / non-null values",
-                            ("By default, non-null values are counted.\n"
+                            ("By default, only non-null values are counted.\n"
                              "This can be changed through ScalarAggregateOptions."),
                             {"array"},
                             "ScalarAggregateOptions"};
 
-const FunctionDoc sum_doc{"Sum values of a numeric array",
-                          ("Null values are ignored. Minimum count of non-NA\n"
-                           "values can be set and NAN is returned if too "
-                           "few are present.\n"
-                           "This can be changed through ScalarAggregateOptions."),
-                          {"array"},
-                          "ScalarAggregateOptions"};
+const FunctionDoc sum_doc{
+    "Sum values of a numeric array",
```

Comment:
```suggestion
    "Compute the sum values of a numeric array",
```

Reply:
it's a bit of a nit for consistency with `Compute the mean`.

Reply:
Yep, thanks!